### DISCUSSION TITLE
Teamleiter darf Teamsbearbeiten

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#3009] [Questions & Answers]
Guten Morgen,

bei uns im Unternehmen ist es nötig das der Teamleiter auch seinen Teams weitere Teammitglieder zuordnet, dafür habe ich die entsprechenden Berechtigungen der Rolle Teamleiter zugeordnet. Teams können editiert werden, leider kann er nur neue Nutzer zuordnen die bereits in Projekten vorhanden sind bei denen er Teamleiter ist. Neue Benutzer kann er leider nicht hinzufügen. 
Sobald ich ein Team anlege in denen alle Benutzer vorhanden sind und der Teamleiter auch die Rolle Teamleiter hat funktioniert es, leider kann er dann natürlich auch unter Berichte die Zeiterfassungen aller Benutzer sehen. 
Übersehe ich etwas bei der Rechtevergabe ?

Vielen Dank für die Hilfe.

**Comment by kevinpapst** [comment]
> Übersehe ich etwas bei der Rechtevergabe ?

Nein. Was ihr wollt geht momentan nicht und ist inhaltlich auch eher ungewöhnlich. 
Die meisten anderen Unternehmen würden genau das unter allen Umständen verbieten wollen.

**Comment by jens-2305** [comment]
Ok danke, schade, ich hatte gehofft das ein Teamleiter auch seine Teams vergrößern kann.

**Reply by kevinpapst** [reply]
Aktuell nicht. Aber wenn Euer Unternehmen groß genug ist um solche Anforderungen zu haben, kann es mich gerne mit Anpassungen beauftragen.
